packages/flutter/test/widgets/modal_barrier_test.dart

```diff
@@ -77,7 +78,32 @@ void main() {
     await tester.pump(const Duration(seconds: 1)); // end transition
 
     // Tap on the barrier to dismiss it
-    await tester.tap(find.byKey(const ValueKey<String>('barrier')));
+    await tester.press(find.byKey(const ValueKey<String>('barrier')));
+    await tester.pump(); // begin transition
+    await tester.pump(const Duration(seconds: 1)); // end transition
```

Comment:
Curious nit: Can these two lines [82-83] be replaced by a single call to `pumpAndSettle`?